includes/polling/mib/ceragon-mib.inc.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
We've recently added support for wireless sensors. this looks like rx/tx rate which isn't currently supported but it makes more sense to wait for that to be added to the core code than adding this old work around and us removing it later.

@murrant any thoughts?